segmentation/requirements.txt

```diff
@@ -0,0 +1,8 @@
+albumentations==1.3.0
+matplotlib==3.6.3
+numpy==1.24.1
+opencv_python==4.7.0.68
+scipy==1.9.1
+segmentation_models_pytorch==0.3.2
+torch==1.13.1
+tqdm==4.64.1
```

Comment:
TODO: for jonas. Right before merge, add link to this notebook in the table of contents in Readme

Reply:
TODO after merge (to avoid conflict): update in [ci.yml](https://github.com/cleanlab/examples/blob/master/.github/workflows/ci.yml) 
```
ignore_paths: 'contrib,fine_tune_LLM,segmentation'
```